unit-tests/py/ac.py

```diff
@@ -0,0 +1,31 @@
+import pyrealsense2 as rs
+from time import sleep
+
+status_list = []
+
+# Functions for using status list
+def status_list_empty():
+    global status_list
+    return not status_list
+
+def reset_status_list():
+    global status_list
+    status_list = []
+
+# Call back function for testing status sequences
+def list_status_cb( status ):
+    global status_list
+    status_list.append(status)
+
+# Removes irrelevant statuses from status list
+def trim_irrelevant_statuses(irrelevant_statuses):
+    global status_list
+    for irrelevant_status in irrelevant_statuses:
+        while irrelevant_status in status_list:
+            status_list.remove(irrelevant_status)
+
+# Waits for calibration to end by checking last status call-back
+def wait_for_calibration():
+    global status_list
+    while status_list[-1] != rs.calibration_status.successful and status_list[-1] != rs.calibration_status.failed:
+        sleep(1)
```

Comment:
Can we at least return the final status, or a boolean whether the test succeeded or not?
The caller may choose to `require` that it succeeded...

Reply:
sure